File: javaEpam/Lab7/src/Crocodile.java
public class Crocodile extends Reptile {
    private String skinTone;

    public Crocodile(String name, String size, int weight, String skinTone) {
        super(name, size, weight);
        this.skinTone = skinTone;
    };

    public String getHeaders() {
        return super.getHeaders() + ',' + "skinTone";
    }

    public String toCSV() {
        return super.toCSV() + ',' + this.skinTone;
    }
}

File: python/lab_11/crocodile.py
from reptile import Reptile

class Crocodile(Reptile):
  skinTone = ""

  def __init__(self, name=None, size=None, weight=None, skinTone=None):
    super().__init__(name, size, weight)
    self.skinTone = skinTone or "dark"
